contracts/README.md

```diff
@@ -1,5 +1,10 @@
 # Bonsai Solidity Contracts
 
-This folder contains the Solidity contract for deploying a Bonsai application through the Ethereum relay. 
+This folder contains the Solidity contract for deploying a [Bonsai] application through the Ethereum relay. 
 
-The contract demonstrates one pattern for offloading the computation of an expensive or difficult to implement function to a RISC Zero guest running on Bonsai.
\ No newline at end of file
+The contract demonstrates one pattern for offloading the computation of an expensive or difficult to implement function to a [RISC Zero] [guest] running on Bonsai.
+
+[Bonsai]: https://dev.bonsai.xyz
+[offloading the computation]: https://twitter.com/RiscZero/status/1677316664772132864
+[RISC Zero]: https://risczero.com
+[guest]: ../methods/guest/src/bin/
```

Comment:
Not blocking and a bit of a point of opinion, but I think we should use a full link rather than a relative link to point to code